src/layouts/MainLayout.vue

```diff
@@ -58,10 +68,21 @@ export default defineComponent({
       lang,
       global: global.data,
       util,
-      autoLaunch: false
+      autoLaunch: false,
+      nodeName: ''
     }
   },
-
-  methods: {}
+  mounted() {
+    this.nodeNameChanger()
+  },
+  methods: {
+    async nodeNameChanger() {
+      let _this = this
```

Comment:
i cheated by using `_this`. 
maybe fat arrows may solve the `this scope` issue.

Reply:
Yes, use fat arrows instead.